host/4/mariner/dotnet/dotnet-isolated/start.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+export DOTNET_USE_POLLING_FILE_WATCHER=true
+
+if [ -z $PORT ]; then
+  export ASPNETCORE_URLS=http://*:80
```

Comment:
Since we don't install the appservice remote debugging tools to this image we don't need start.sh or sshd_config I believe.  We don't copy them into the image and it works.  So please remove them from this directory.

Reply:
Removed